saleor/core/utils/url.py

```diff
@@ -33,3 +36,17 @@ def prepare_url(params: str, redirect_url: str) -> str:
     split_url = urlsplit(redirect_url)
     split_url = split_url._replace(query=params)
     return split_url.geturl()
+
+
+class APPURLValidator(URLValidator):
```

Comment:
I think it should say: `AppURLValidator`

Reply:
I also think this validator could be part of the `saleor/app` module. It's specific to apps; also it is used only in app-related modules.